multi_dimensional_flavors/shorebird.yaml

```diff
@@ -0,0 +1,23 @@
+# This file is used to configure the Shorebird updater used by your app.
+# Learn more at https://docs.shorebird.dev
+# This file should be checked into version control.
+
+# This is the unique identifier assigned to your app.
+# Your app_id is not a secret and is just used to identify your app
+# when requesting patches from Shorebird's servers.
+app_id: 5171ee3d-1a4d-4bdd-91c3-64cef78fe41c
+flavors:
+  internal: 5171ee3d-1a4d-4bdd-91c3-64cef78fe41c
```

Comment:
I don't think this is a valid configuration/app so we're unnecessarily creating an app which can never be used.

Reply:
Makes sense, there is even this issue here to track this: https://github.com/shorebirdtech/shorebird/issues/1961

Good catch, I am going to remove the invalid apps from the example